Tests/base/NSString/test09.m

```diff
@@ -23,35 +29,60 @@ int main (int argc, const char * argv[])
   NSString *urlEncodedString =
     @"Only%20alphabetic%20characters%20should%20be%20allowed%20and%20not%20encoded%2E%20%21%40%23%24%25%5E%26%2A%28%29%5F%2B%2D%3D";
   NSCharacterSet *allowedCharacterSet = [NSCharacterSet alphanumericCharacterSet];
-  PASS(testUrlCharacterSetEncoding(urlDecodedString, urlEncodedString, allowedCharacterSet), "alphanumericCharacterSet");  
+  PASS(testUrlCharacterSetEncoding(urlDecodedString, urlEncodedString, allowedCharacterSet), "alphanumericCharacterSet");
```

Comment:
All the tests here could be using the PASS_EQUAL macro to display the strings when the test is failing.
Or you could move both the tests into the helper method

Reply:
Yes, there is a lot of room for improvement, but since this is my first MR, I tried to adhere to the style in which this test was written earlier.